### PR TITLE
Connection to firebase

### LINKED ISSUE #8: Connect to Firebase
### Tasks
- [x] Connect to Firebase in Project
- [x] Create User Login Validation
- [x] Create User SignUp

### PR DESCRIPTION
**Pull request by ecruz17**
# Description

- Configured and connected React Native Project to Firebase Project
- Created methods to SignIn users with their email and password
- Created LoadingCircle Component
- Created Login Screen

## Screenshots

SignUp

### CODE REVIEW

**Review by Eduardoespl — APPROVED** [review]
🚀

File: reservation/android/app/google-services.json
Comment (BrandonVG) [inline reservation/android/app/google-services.json:1]: You should hide this file, or the file's content

**Review by BrandonVG — APPROVED** [review]
Nice job!

**Review by luisgarciaes — APPROVED** [review]
LGTM!

File: reservation/android/app/google-services.json
Comment (ecruz17) [reply reservation/android/app/google-services.json:1]: thx m8, will do!

**Review by FranciscoSenestrari — APPROVED** [review]

**Pull request merged**